docs/about.md

```diff
@@ -34,3 +34,22 @@ Talawa’s desired main features are:
 1. Donation acceptance
 1. Service provider level multi-organization capability
 1. Plugin support for administration services such as billing
+
+### Technologies Used
+
+Technology plays an increasingly significant role in improving access to education for people living in impoverished areas and developing countries. Some of the technologies that are being used by Talawa are:
+
+
+1. Front-end Technologies: HTML, CSS, Flutter, TypeScript and JavaScript is used to create the website's front-end user interface.
```

Comment:
We have a web portal and mobile app. The technologies for each specifically need to be mentioned.

Reply:
Could you please help me to understand this thing a little briefly?

Reply:
For example split it somehow between web front end technologies and mobile app front end technologies.

Reply:
Done